### DISCUSSION TITLE
multi directory

### DISCUSSION THREAD

**Discussion by NookieGrey** [community/community#21537] [Other Feature Feedback, Questions, & Ideas]
<p>I cloned project js-xlsx and set up github pages.</p>
<p>When I open in browser <a href="https://nugzargaguliya.github.io/js-xlsx/" rel="nofollow noopener">/js-xlsx/</a> it loads /index.html.</p>
<p>When I open <a href="https://nugzargaguliya.github.io/js-xlsx/demos/" rel="nofollow noopener">/js-xlsx/demos/</a> it loads /demos/README.md</p>
<p>But I can’t open third folder <a href="https://nugzargaguliya.github.io/js-xlsx/demos/react/" rel="nofollow noopener">/js-xlsx/demos/react/</a>. It returns 404 page.</p>
<p>My repository is <a href="https://github.com/NugzarGaguliya/js-xlsx" rel="nofollow noopener">https://github.com/NugzarGaguliya/js-xlsx</a></p>
<p>Is it any settings or forbiddance for multi directories?</p>

**Comment by nadiajoyce** [comment, marked as answer]
<p><a class="mention" href="/nugzargaguliya">@nugzargaguliya</a> I was able to open that folder just fine. Are you still having this issue? If so, you might want to try reaching out to <a>private GitHub Support</a> so we can dig into your GitHub account in a little more detail.</p>

**Comment by NookieGrey** [comment]
<p>Thank you. Everything is working fine. I think the problem was in the cache.</p>
